### DISCUSSION TITLE
no executable git for consulo ide

### DISCUSSION THREAD

**Discussion by NwosuTy** [consulo/UnityEditorConsuloPlugin#29] [Q&A]
![Lets Fight - SampleScene - PC, Mac   Linux Standalone - Unity 2020 1 8f1 Personal _DX11_ 25-Nov-21 4_34_59 PM](https://user-images.githubusercontent.com/94308549/143469482-6fe6de78-9123-4e52-ac91-d16f275f35a7.png)

Having problems with adding the unity plugin could you please help

**Comment by VISTALL** [comment]
Hello.  Due plugin  source published at github (as git repo). Git tool required https://git-scm.com/downloads

Consulo not require this plugin. If you first time using Consulo - you can ignore for now this plugin. Debugger, project analyze works without Unity communications.

**Reply by NwosuTy** [reply]
Yeah ive downloaded that already but it still shows the same error

i was wondering if i needed to download this plugin https://github.com/consulo/UnityEditorConsuloPlugin

**Reply by VISTALL** [reply]
Install it, and restart Unity. It's Unity problem. It use git for repo dowloading.  Nope - just add it to manifest. Also you can download it, and place inside project (just sources), or choose downloaded directory as local package from unity

Also you can drop this project inside Packages dir, and Unity will load it too

**Reply by NwosuTy** [reply]
sorry could you explain how i could add it to manifest

ive downloaded and installed git already

**Reply by VISTALL** [reply]
Not git. UnityEditorConsuloPlugin must be inside manifest (you already added lines for it - see readme of this project).
It Unity not saw git. 

Check this forum post at unity

https://forum.unity.com/threads/no-git-executable-was-found-please-install-git-on-your-system-and-restart-unity.730511/

**Reply by NwosuTy** [reply]
aii thank you lemme try that and get back to you

**Reply by NwosuTy** [reply]
it did work thank you
